app/src/main/java/com/android/streetworkapp/ui/progress/Gamification.kt

```diff
@@ -29,7 +38,7 @@ fun updateAndDisplayPoints(
     when (result) {
       SnackbarResult.Dismissed -> {}
       SnackbarResult.ActionPerformed -> {
-        navigationActions.navigateTo(Screen.PROGRESSION)
+        navigationActions.navigateTo(TopLevelDestinations.PROGRESSION)
```

Comment:
During the implementation of this Pull Request, I manage to find a bug while using the Screen.Progression: Clicking on the map icon redirected to the progression screen. To avoid this issue, i added this TopLevelDestination. This is the reason why I think it is better to let is like this.

Reply:
fair enough